### ISSUE TITLE
Webfront does not open (and The configured user limit (128) on the number of inotify instances has been reached issue) on Linux for 2022.10.17.2

### ISSUE THREAD

**Issue by DudeShift** [RaidMax/IW4M-Admin#270]
I know this is two issues but temporary fixed one but may be related to another. 

Using Ubuntu 18.04 Linux 

The Webfront worked for me one time while I was figuring out the server log location was. Once I was able to get the correct server log location, I got a `The configured user limit (128) on the number of inotify instances has been reached issue` error when running ./StartIW4MAdmin.sh

I temporary fixed this by a quick google search and [using `echo fs.inotify.max_user_instances=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p` as suggested here](https://anthonyliriano.com/2021/04/08/21-configured-user-limit-128-on-the-number-of-inotify-instances-has-been-reached.html).

Now however the webfront won't open at, tried different browsers and devices. In game commands like !owner !rt work though.

With `nmap -p 1624 {Server IP Here}`:
```
PORT      STATE   SERVICE
1624/tcp open  udp-sr-port
```

Below is the log file from IW4MAdmin (Clients are from Bot Warfare mod with custom names in userraw):
```
[2022-10-20 06:22:43.189  INF] Begin IW4MAdmin startup. Version is 2022.10.17.2 []
[2022-10-20 06:22:56.058  WRN] Id and SubscriptionId must be provided to attempt loading remote assemblies/scripts
[2022-10-20 06:22:56.092  WRN] Id and SubscriptionId must be provided to attempt loading remote assemblies/scripts
[2022-10-20 06:22:57.277  INF] Beginning database migration sync
[2022-10-20 06:22:58.805  WRN] Sensitive data logging is enabled. Log entries and exception messages may include sensitive application data; this mode should only be enabled during development.
[2022-10-20 06:23:00.073  INF] Finished database migration sync
[2022-10-20 06:23:01.434  INF] Game Interface Startup
[2022-10-20 06:23:01.634  INF] Loaded 0 ids into whitelist
[2022-10-20 06:23:01.951  INF] Loaded 0 banned subnets
[2022-10-20 06:23:02.323 {Server IP Here}:28960 INF] Beginning server communication initialization
[2022-10-20 06:23:04.538 {Server IP Here}:28960 INF] Generated game log path is /root/iw4x-docker/data/userraw/logs/gameserver_mp.log
[2022-10-20 06:23:04.558 ServerNameHere INF] Finishing initialization and now monitoring [ServerNameHere]
[2022-10-20 06:23:05.329 {Server IP Here}:28960 INF] GSC Integration enabled = false
[2022-10-20 06:23:07.103 {Server IP Here}:28960 INF] Detected PreConnect for [Name=Biscuits^7, NetworkId=2AF8039E, IP=255.255.255.255, ClientSlot=0] from "Status"
[2022-10-20 06:23:07.436 {Server IP Here}:28960 INF] Detected PreConnect for [Name=myself^7, NetworkId=70588C52, IP=255.255.255.255, ClientSlot=1] from "Status"
[2022-10-20 06:23:07.452 {Server IP Here}:28960 INF] Detected PreConnect for [Name=Dick_chopper^7, NetworkId=FFFFFFFFB17C07ED, IP=255.255.255.255, ClientSlot=2] from "Status"
[2022-10-20 06:23:07.473 {Server IP Here}:28960 INF] Detected PreConnect for [Name=TastyGuacamole^7, NetworkId=FFFFFFFF84A60B5D, IP=255.255.255.255, ClientSlot=3] from "Status"
[2022-10-20 06:23:07.480 {Server IP Here}:28960 INF] Client [Name=Biscuits^7, NetworkId=2AF8039E, IP=255.255.255.255, ClientSlot=0] is joining the game from Status
[2022-10-20 06:23:07.480 {Server IP Here}:28960 INF] Received ip from client [Name=Biscuits^7, NetworkId=2AF8039E, IP=255.255.255.255, ClientSlot=0]
[2022-10-20 06:23:07.480 {Server IP Here}:28960 INF] Client [Name=Dick_chopper^7, NetworkId=FFFFFFFFB17C07ED, IP=255.255.255.255, ClientSlot=2] is joining the game from Status
[2022-10-20 06:23:07.481 {Server IP Here}:28960 INF] Received ip from client [Name=Dick_chopper^7, NetworkId=FFFFFFFFB17C07ED, IP=255.255.255.255, ClientSlot=2]
[2022-10-20 06:23:07.482 {Server IP Here}:28960 INF] Client [Name=myself^7, NetworkId=70588C52, IP=255.255.255.255, ClientSlot=1] is joining the game from Status
[2022-10-20 06:23:07.483 {Server IP Here}:28960 INF] Received ip from client [Name=myself^7, NetworkId=70588C52, IP=255.255.255.255, ClientSlot=1]
[2022-10-20 06:23:07.491 {Server IP Here}:28960 INF] Client [Name=TastyGuacamole^7, NetworkId=FFFFFFFF84A60B5D, IP=255.255.255.255, ClientSlot=3] is joining the game from Status
[2022-10-20 06:23:07.491 {Server IP Here}:28960 INF] Received ip from client [Name=TastyGuacamole^7, NetworkId=FFFFFFFF84A60B5D, IP=255.255.255.255, ClientSlot=3]
[2022-10-20 06:23:07.496 {Server IP Here}:28960 INF] Detected PreConnect for [Name=Blueberrythebea^7, NetworkId=22F17DBB, IP=255.255.255.255, ClientSlot=4] from "Status"
[2022-10-20 06:23:07.507 {Server IP Here}:28960 INF] Client [Name=Blueberrythebea^7, NetworkId=22F17DBB, IP=255.255.255.255, ClientSlot=4] is joining the game from Status
[2022-10-20 06:23:07.507 {Server IP Here}:28960 INF] Received ip from client [Name=Blueberrythebea^7, NetworkId=22F17DBB, IP=255.255.255.255, ClientSlot=4]
[2022-10-20 06:23:07.509 {Server IP Here}:28960 INF] Detected PreConnect for [Name=CaptainSombrero^7, NetworkId=FFFFFFFFDE705781, IP=255.255.255.255, ClientSlot=5] from "Status"
[2022-10-20 06:23:07.552 {Server IP Here}:28960 INF] Client [Name=CaptainSombrero^7, NetworkId=FFFFFFFFDE705781, IP=255.255.255.255, ClientSlot=5] is joining the game from Status
[2022-10-20 06:23:07.552 {Server IP Here}:28960 INF] Received ip from client [Name=CaptainSombrero^7, NetworkId=FFFFFFFFDE705781, IP=255.255.255.255, ClientSlot=5]
[2022-10-20 06:23:07.554 {Server IP Here}:28960 INF] Detected PreConnect for [Name=ElbowDrop^7, NetworkId=FFFFFFFF8A339938, IP=255.255.255.255, ClientSlot=6] from "Status"
[2022-10-20 06:23:07.582 {Server IP Here}:28960 INF] Client [Name=ElbowDrop^7, NetworkId=FFFFFFFF8A339938, IP=255.255.255.255, ClientSlot=6] is joining the game from Status
[2022-10-20 06:23:07.582 {Server IP Here}:28960 INF] Received ip from client [Name=ElbowDrop^7, NetworkId=FFFFFFFF8A339938, IP=255.255.255.255, ClientSlot=6]
[2022-10-20 06:23:07.584 {Server IP Here}:28960 INF] Detected PreConnect for [Name=runNINGshRIMP^7, NetworkId=10F8FA32, IP=255.255.255.255, ClientSlot=7] from "Status"
[2022-10-20 06:23:08.065 {Server IP Here}:28960 INF] Client [Name=runNINGshRIMP^7, NetworkId=10F8FA32, IP=255.255.255.255, ClientSlot=7] is joining the game from Status
[2022-10-20 06:23:08.065 {Server IP Here}:28960 INF] Received ip from client [Name=runNINGshRIMP^7, NetworkId=10F8FA32, IP=255.255.255.255, ClientSlot=7]
```

**Comment by RaidMax** [comment]
Pretty certain this is an environment issue. 
We have plenty of users running on Linux including myself. 
It looks like you are running through docker, have you mapped the docker container port to your machine port?
What does your `WebfrontBindUrl` look like in `IW4MAdminSettings.json`? 
Also probably the most effective way to troubleshoot is via our Discord, as this does not appear to be a code issue.

**Comment by DudeShift** [comment]
Upgraded / Installed Ubuntu 22.04 as I overlooked that .NET Core 6.0.x was required which is only available on 22.04. No problems now.
